src/SFA.DAS.Apprenticeships.Command.UnitTests/ApprenticeshipDomainModelTestHelper.cs

```diff
@@ -9,12 +11,46 @@ public static class ApprenticeshipDomainModelTestHelper
 {
     private static readonly Fixture _fixture = new();
 
-    public static void AddEpisode(ApprenticeshipDomainModel apprenticeship, DateTime? startDate = null, DateTime? endDate = null)
+    // If this method isn't a sign that we need to refactor this project then I don't know what is
+    internal static ApprenticeshipDomainModel CreateBasicTestModel()
```

Comment:
These helper methods were something Dina was very pro, maybe something to discuss if we want to change the approach?

Reply:
Not so much the helper methods, but the way the domain models/entity models are constructed. One for the tech debt I think